README.md

```diff
@@ -35,7 +35,7 @@ inference.
     - [Person Re-Identification](pytorch_toolkit/object_reidentification/person_reidentification)
     - [Vehicle Re-Identification](pytorch_toolkit/object_reidentification/vehicle_reidentification)
   * [Human Pose Estimation](pytorch_toolkit/human_pose_estimation)
-  * [Instance Segmentation](pytorch_toolkit/instance_segmentation)
+  * [Instance Segmentation](pytorch_toolkit/instance_segmentation_2)
```

Comment:
Do we need these `_2` suffixes as long as we have only one training engine now?

Reply:
removed